crates/solver/src/settlement.rs

```diff
@@ -248,6 +248,11 @@ impl Settlement {
         }
     }
 
+    /// Creates an empty, but valid, settlement.
+    pub fn empty() -> Self {
+        Self::default()
+    }
```

Comment:
Interesting. I thought `Settlement::empty()` made it clearer at the callsite that we want an empty settlement. The fact that the `default` is also an empty settlement is a coincidence but not semantically required (i.e. we can change the default trade two orders in a perfect CoW, but the empty will always be the empty settlement).

I don't feel strongly about this at all, just thought it would be easier to read where it is called.

I will remove since others tend to agree that it doesn't help.

Reply:
You could document that default is empty and make that part of the contract. Depends on whether you say default is supposed to be a completely arbitrary value. In practice it often means empty or zero. In those cases I don't think a separate function is needed. I don't have a strong opinion here.

Reply:
Changed back to `::default()`.